roboteam_ai/src/utilities/OffensiveCoach.cpp

```diff
@@ -0,0 +1,317 @@
+//
+// Created by robzelluf on 3/8/19.
+//
+
+#include <roboteam_ai/src/interface/widget.h>
+#include <roboteam_ai/src/interface/drawer.h>
+#include "OffensiveCoach.h"
+
+namespace rtt {
+namespace ai {
+namespace coach {
+
```

Comment:
the 2 max variables should be constants, IMO

Reply:
Just discussed this with @ThijsLuttikhuis today, in our opinion not everything should just go to the constants file. These to variables are very specific to the offensive coach, so I think it should be fine if they're located here.

Reply:
Since they are very specific to the OffensiveCoach, I think it should be fine that they stay here